internal/plugins/ansible/v1/scaffolds/internal/templates/config/crd/crd.go

```diff
@@ -40,22 +37,18 @@ func (f *CRD) SetTemplateDefaults() error {
 	}
 	f.Path = f.Resource.Replacer().Replace(f.Path)
 
-	f.IfExistsAction = file.Error
+	f.IfExistsAction = machinery.Error
 
-	if f.CRDVersion == "" {
-		f.CRDVersion = "v1"
-	} else if f.CRDVersion != "v1" && f.CRDVersion != "v1beta1" {
-		return errors.New("the CRD version value must be either 'v1' or 'v1beta1'")
-	}
 	f.TemplateBody = fmt.Sprintf(crdTemplate,
 		text.Indent(openAPIV3SchemaTemplate, "    "),
 		text.Indent(openAPIV3SchemaTemplate, "      "),
```

Comment:
Can we add a comment here why it is required to make it easier for others in the future?

Reply:
What are you exactly talking about? The only change I did in this file was fixing the import and using directly the CRDVersion from the resource and not passing it manually.

Reply:
Why we need to use `openAPIV3SchemaTemplate, "      "` and `openAPIV3SchemaTemplate, "    "`?

Reply:
I have no idea what that is required for.